<ui version="4.0" >
 <class>addBuddyDialogClass</class>
 <widget class="QDialog" name="addBuddyDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>104</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>104</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>addBuddyDialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>7</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>7</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Local name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="nickEdit" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="maxLength" >
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Group:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QComboBox" name="groupList" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>121</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="addButton" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text" >
      <string>Add</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/apply.png</normaloff>:/icons/crystal_project/apply.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../../../qutim.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>addBuddyDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>159</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel" >
     <x>84</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
